src/os/bluestore/BlueFS.cc

```diff
@@ -2763,9 +2760,10 @@ void BlueFS::_compact_log_async_LD_LNF_D() //also locks FW for new_writer
 
   // 1.1 allocate new log extents and store them at fnode_tail
   File *log_file = log.writer->file.get();
+  _maybe_extend_log(log.t.expected_size(), true);
```

Comment:
`log.t` is empty here. We start compacting right after flushing the log. And as a reaction to direct request `compact()`, but this is rare.

And even if there was some ops accumulated in log.t, there is no reason to prepare size for it in the log that is in process of switching. The next log is the one that will possibly need to accommodate size for it.